tests/test_image_plugin.py

```diff
@@ -19,3 +19,8 @@ def test_image(snapshot_png):
         b"RK5CYII="
     )
     assert actual == snapshot_png
+
+
+def test_raises_error_for_unserializable_data(snapshot_png):
+    with pytest.raises(ValueError):
+        assert "not a byte string" == snapshot_png
```

Comment:
not sure if it's best to throw an error here, or consider this a failed snapshot.

Reply:
We could wrap the serialization with an error handler and compare the raw values if it fails